spec/cluster_holding_spec.rb

```diff
@@ -78,6 +78,20 @@ def new_submission(holding, date: Date.today)
         eq(h2.date_received)
     end
 
+    it "updates multiple existing holdings if cluster.large?" do
+      c.ocns << Services.large_clusters.ocns.first
+      c.save
+      described_class.new(h).cluster
+      described_class.new(h.clone).cluster
+      h2.date_received = Date.today
+      h2.uuid = SecureRandom.uuid
+      described_class.new(h2).update
+      cluster = Cluster.first
+      expect(cluster.holdings.first.date_received).to eq(h2.date_received)
```

Comment:
This spec confuses me because it doesn't seem to take into account that large clusters should skip duplicative holdings. There's only one holding to update. Maybe this is an artifact of the order the tests were written in? If nothing else, it might be worth a comment or two to explain that the observed behavior is a side effect of only retaining one holding record.

Reply:
See above.